src/main/resources/edu/cornell/kfs/fp/cu-ojb-fp.xml

```diff
@@ -1101,5 +1101,478 @@
     <field-descriptor name="objectId" column="OBJ_ID" jdbc-type="VARCHAR" index="true"/>
     <field-descriptor name="noteText" column="ACH_NTE_TXT" jdbc-type="VARCHAR"/>
   </class-descriptor>
+  
+  <class-descriptor
+	class="edu.cornell.kfs.fp.document.CorporateBilledCorporatePaidDocument"
+	table="CU_FP_CBCP_DOC_T">
```

Comment:
If you want these descriptors to use tabs instead of spaces, then please update the "class-descriptior" tag opening above to also use tabs.